### ISSUE TITLE
Design problems on mobile

### ISSUE THREAD

**Issue by andywarburton** [szabodanika/click-clack-public#3]
![340DF0F0-D480-4287-A23F-439CFCE16173](https://user-images.githubusercontent.com/507878/85427514-23d5ae80-b57c-11ea-9fb1-ac1c8f1ddfd6.png)

On mobile the heading is all bunched up into a tiny column and the floating header overlaps the top of the content when you scroll to the top of the page so you can’t see the whole photo.

(Iphone 11 pro)

**Comment by szabodanika** [comment]
Yeah mobile optimization is terrible atm, the new fixed header only made it worse. There will be an update focusing on mobile optimization in the upcoming days

**Comment by szabodanika** [comment]
I made a couple improvements here and there in the past couple updates but @alfredcheuk will now review the entire frontent and we'll see a lot of improvement. Good luck!
